omod/src/main/java/org/openmrs/module/adminui/page/controller/systemadmin/accounts/AccountPageController.java

```diff
@@ -109,9 +117,52 @@ public String post(PageModel model, @MethodParam("getAccount") @BindParams Accou
 	                   @SpringBean("adminService") AdministrationService administrationService,
 	                   @SpringBean("adminUiAccountValidator") AdminUiAccountValidator accountValidator,
 	                   @SpringBean("providerManagementService") ProviderManagementService providerManagementService,
+					   @SpringBean("appFrameworkService") AppFrameworkService appFrameworkService,
 	                   HttpServletRequest request, UiUtils uu) throws IOException {
 		
 		Errors errors = new BeanPropertyBindingResult(account, "account");
+
+        List<Extension> customUserFragments = appFrameworkService.getExtensionsForCurrentUser("createUser.manageAccountUserFragments");
+        Map<String, String[]> parameterMap = request.getParameterMap();
+        for(Extension ext : customUserFragments) {
+            if (StringUtils.equals(ext.getExtensionParams().get("type").toString(), "userProperty")) {
+                String userPropertyName = ext.getExtensionParams().get("userPropertyName").toString();
+                String[] parameterValues = parameterMap.get(userPropertyName);
+                if (parameterValues != null && parameterValues.length > 0) {
+                    if (parameterValues.length > 1) {
+                        log.warn("Multiple userProperty for a single user type not supported, ignoring extra values");
+                    }
+                    String parameterValue = parameterValues[0];
+                    if (userPropertyName !=null && parameterValue != null) {
```

Comment:
Can we also put space around the above !=

Reply:
changed